applications/RomApplication/python_scripts/ConvectionDiffusionROMsolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import convection_diffusion_base_solver
+import KratosMultiphysics.RomApplication as romapp
+
+def CreateSolver(main_model_part, custom_settings):
+    return ROMSolver(main_model_part, custom_settings)
+
+class ROMSolver(convection_diffusion_base_solver.ConvectionDiffusionBaseSolver):
+    """The stationary class for ROM convection-diffusion solvers.
+
+    Public member variables:
+    stationary_settings -- settings for the implicit dynamic solvers.
+
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, main_model_part, custom_settings):
+        # Set defaults and validate custom settings.
+        self.stationary_settings = KratosMultiphysics.Parameters(r"""{}""")
+
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ROMSolver, self).__init__(main_model_part, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        if (self.settings["element_replace_settings"]["element_name"].GetString() == "EulerianConvDiff"):
+            self.min_buffer_size = 2
+        else:
+            self.min_buffer_size = 1
+
+        KratosMultiphysics.Logger.PrintInfo("::[ROMSolver]:: ", "Construction finished")
+
+    #### Private functions ####
+    @classmethod
+    def GetDefaultSettings(cls):
+        default_settings = KratosMultiphysics.Parameters("""
+        {            
+            "rom_settings": {
+            "nodal_unknowns": [ "TEMPERATURE" ],
+            "number_of_rom_dofs": 3
+            }
+        }
+        """)
+        default_settings.AddMissingParameters(super(ROMSolver,cls).GetDefaultSettings())
+        return default_settings
+
+    def AddVariables(self):
+        super(ROMSolver, self).AddVariables() #Adding nodal area variable
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_AREA)
+
+
+    def _create_solution_scheme(self):
+        #Variable defining the temporal scheme (0: Forward Euler, 1: Backward Euler, 0.5: Crank-Nicolson)
+        self.GetComputingModelPart().ProcessInfo[ConvectionDiffusionApplication.THETA] = 1.0
```

Comment:
Shouldn't these values be controlled by the underlying convection diffusion solver?